### ISSUE TITLE
Feature Request - Ship a release as a flatpak/snap

### ISSUE THREAD

**Issue by SohamG** [Jessecar96/SteamDesktopAuthenticator#506]
Title. I'd love to see a flatpak or snap for this, itll make life for linux'ers much easier. Consider this please

**Comment by dyc3** [comment]
I made a CLI program to replace SteamDesktopAuthenticator on Linux, [steamguard-cli](https://github.com/dyc3/steamguard-cli). It can read and write the same files that SteamDesktopAuthenticator can, so you don't have to set up your mobile authenticator again. It can also handle trade confirmations.

**Comment by SohamG** [comment]
@dyc3 Yes, I use that. Regardless a GUI would've been nice...I guess everythings a cli on linux
